Assets/Project/Scripts/Modules/GamePlayScene/Bottle/DynamicBottleController.cs

```diff
@@ -28,27 +28,72 @@ public class DynamicBottleController : AbstractBottleController
         /// <summary>
         /// 移動開始時の処理
         /// </summary>
-        public event Action OnStartMove;
+        public event Action OnStartMove
+        {
+            add {
+                _onStartMove += value;
+            }
+            remove {
+                _onStartMove -= value;
+            }
+        }
+        private event Action _onStartMove;
 
         /// <summary>
         /// 移動終了時の処理
         /// </summary>
-        public event Action OnEndMove;
+        public event Action OnEndMove
+        {
+            add {
+                _onEndMove += value;
+            }
+            remove {
+                _onEndMove -= value;
+            }
+        }
+        private event Action _onEndMove;
 
         /// <summary>
         /// ホールド開始時の処理
         /// </summary>
-        public event Action OnPressed;
+        public event Action OnPressed
```

Comment:
`OnStartMove` と合わせるなら
```suggestion
        public event Action OnStartPressed
```

Reply:
`Release`と似た理由になるけど、`Press`というジェスチャに紐づいているので、このままが良いです